include/bootchooser.h

```diff
@@ -55,6 +55,28 @@ G_GNUC_WARN_UNUSED_RESULT;
 gboolean r_boot_set_state(RaucSlot *slot, gboolean good, GError **error)
 G_GNUC_WARN_UNUSED_RESULT;
 
+/**
+ * Enable or disable locking of the boot counter.
+ * This variable is not set per slot, but is valid for all slots.
+ *
+ * @param locked TRUE to lock counting, FALSE to unlock it
+ * @param error return location for a GError, or NULL
+ *
+ * @return TRUE if successful, FALSE if failed
+ */
+gboolean r_boot_set_counters_lock(gboolean locked, GError **error);
+
+/**
+ * Check current status of boot counter locking in the bootloader.
+ * This variable is not set per slot, but is valid for all slots.
+ *
+ * @param locked return location for value of lock-counter
```

Comment:
What's `lock-counter` here? So far this is only the name of the 'prevent-late-fallback' strategy.

Reply:
hmhm, to keep it generic and not just to match barebox: it is a related bootloader value
so maybe something like
```suggestions
 * @param locked return location for the current value in the bootloader
```